Applications/PlazaDetails/Reporting.qmd

```diff
@@ -7,24 +7,25 @@ aliases:
 
-Please check that you are logged in to your account for accessing the reporting functionality. Once logged in, select the "Reporting" option from the navbar, that will take you to the screen shown in the figure below:
+Please ensure that the account is logged in to access the reporting functionality. Once logged in, navigate to the "Reporting" option in the navigation bar. This will take users to the screen displayed in the figure below:
 
 ![](../_images/Reporting.png)
 
-If you haven't executed anything yet, there may not be much information displayed on the screen. However, as you start utilizing openEO and the services on the plaza, the screen will populate with relevant data that can be visualized in the following manner:
+If the user has not run any jobs, the screen will display less information. However, if openEO or Algorithm Plaza's services are used, relevant information will be displayed. The screen is divided into two sections: "Personal" and "Service." The "Personal" section provides a detailed overview for:
 
-1. Filtering Reports: You will have the option to filter the reports based on a specific time interval. This allows you to view the openEO usage and credit deductions within a desired timeframe, providing a more targeted analysis.
-2. Credit Usage Display: A section will be available to display the credits used, showcasing both the synchronous and batch job execution methods. This information helps you understand how your credits are being utilized for different types of job executions. Hovering over the credit usage details will provide additional information on the specific service used.
-3. Jobs List: The screen will include a list of the jobs that have been executed. This list provides a comprehensive overview of the services that have been utilized. Each job entry will typically include details such as job ID, execution time, and any relevant metadata associated with the job.
+1. Filtering Reports: Users will have the option to filter the reports based on a specific time interval. This allows them to view the openEO usage and credit deductions within a desired timeframe, providing a more targeted analysis.
+2. Credit Usage Display: A section will be available to display the credits used, showcasing both the synchronous and batch job methods. This information helps in understanding how credits are being utilized for different types of job. Hovering over the credit usage details will provide additional information on the specific service used.
+3. Jobs List: The screen will include a list of the jobs that has been run. This list provides a comprehensive overview of the services that have been utilized. Each job entry will typically include details such as job ID, time to run the job, and any relevant metadata associated with the job.
```

Comment:
The ... screen includes a comprehensive list of all past jobs and used services.

The job duration